### ISSUE TITLE
VideoClassifier

### ISSUE THREAD

**Issue by jhfjhfj1** [keras-team/autokeras#137]
### Suggested Name

VideoClassifier

### Task Description

The training instances should be video clips. The target is a single label for each instance.

### Evaluation Metrics

Accuracy

### Benchmark Datasets

[UCF101 Action Recognition Data Set](http://crcv.ucf.edu/data/UCF101.php)

### Solution

Two-Stream Inflated 3D ConvNet(I3D) looks like the state of the art solution toward video classification. [I3D](https://arxiv.org/pdf/1705.07750.pdf)  With pretrained on [kinetics dataset](https://deepmind.com/research/open-source/open-source-datasets/kinetics/)  can do 80.9% on HMDB-51 and 98.0% on UCF-101
